dataland-qa-service/src/main/resources/application.properties

```diff
@@ -1,14 +1,34 @@
 springdoc.show-actuator=true
+# TODO swagger ui stuff, do we want to keep that?
+springdoc.swagger-ui.oauth2RedirectUrl=https://${PROXY_PRIMARY_URL:local-dev.dataland.com}/documents/swagger-ui/oauth2-redirect.html
+springdoc.swagger-ui.oauth.use-pkce-with-authorization-code-grant=true
+springdoc.swagger-ui.oauth.client-id=dataland-public
+
 management.endpoints.web.exposure.include=health, info
 management.endpoint.health.show-details=always
 management.info.git.mode=FULL
 server.tomcat.threads.min-spare=100
 server.shutdown=graceful
 server.servlet.context-path=/qa
+spring.security.oauth2.resourceserver.jwt.issuer-uri=https://${PROXY_PRIMARY_URL:local-dev.dataland.com}/keycloak/realms/datalandsecurity
+spring.security.oauth2.resourceserver.jwt.jwk-set-uri=http://keycloak:8080/keycloak/realms/datalandsecurity/protocol/openid-connect/certs
+dataland.authorization.publiclinks=/actuator/health,/actuator/health/ping,/actuator/info,/swagger-ui/**,/v3/api-docs/**
+
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.username=qa_service
+spring.datasource.password=${QA_SERVICE_DB_PASSWORD}
+spring.datasource.url=jdbc:postgresql://qa-service-db:5432/qa_service
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.show-sql=false
+spring.jpa.hibernate.ddl-auto=update
 spring.jackson.deserialization.fail-on-unknown-properties=true
 server.error.whitelabel.enabled=false
 spring.mvc.throw-exception-if-no-handler-found=true
 spring.web.resources.add-mappings=false
+spring.jpa.open-in-view=false
+spring.datasource.hikari.maximum-pool-size=50
+
+dataland.expose-error-stack-trace-to-api=false
 
```

Comment:
While I was reviewing this, I had the following thought:

Most of our props in the `application.properties` are same across all our microservices.
Therefore we have a lot of duplicate code.

Proposal: We could introduce a "application.properties.template" which summarizes the duplicate settings we want in all services, and extend them by microservice-individual props for each new service.

What do you think?  This could be a story for itself of course and does not need to be done in this feature branch here.

Reply:
If its worth the effort, I'd even like a task to autogenerate a whole spring project

Reply:
Ok do you want to create a story for this,   or just present the idea in the next refinement?

I don't see this happening until the launch in October anyways.

As soon as you have created a story, or taken this as homework to present the idea in the next refinement (I will be on vacation), I will close this conversation.

Reply:
Florian hat sich notiert, diese Idee im nächsten refinement meeting in die Runde zu werfen.